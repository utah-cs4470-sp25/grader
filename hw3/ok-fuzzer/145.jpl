read image "" to x
time show [[[[[[q]]]]]]
