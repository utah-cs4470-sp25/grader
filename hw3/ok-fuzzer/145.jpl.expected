(ReadCmd "" (VarLValue x))
(TimeCmd (ShowCmd (ArrayLiteralExpr (ArrayLiteralExpr (ArrayLiteralExpr (ArrayLiteralExpr (ArrayLiteralExpr (ArrayLiteralExpr (VarExpr q)))))))))
